docs/findings/el7.md

```diff
@@ -289,3 +295,14 @@ Typically caused when a user is created via a service/process like `cloud-init`:
 Only valid in environments where individually-defined DNS servers are not highly-available.
 
 When deployed into environments where DNS is provided through a highly-available service with a highly-available service-name, only one DNS server will be configured into the host's `/etc/resolv.conf` &ndash; typically by way of a DHCP option-set.
+
+# The OS Must Elevate The SELinux Context When An Administrator Calls The Sudo Command
+
+**Conditionally Valid:**
+
+Implementation of this finding's technical controls changes how the `sudo` commands are executed. Some EL7 tooling (at least one third-party authentication subsystem is known to break under this new control) is incompatible with implementing this control. For systems where this control breaks functionality, and must be disabled, this will be a valid finding that should be included in any exception documentation and associated organizational-processes. Otherwise the system should be configured to meet this control.
+
+**Further Notes:**
+
+1. Implementing this control can have significant user-education requirements and can also adversely-impact legacy automation. While these _should_ be non-fatal problems &ndash; only requiring user-education or fine-tuning of legacy automation, the control still should be implemented.
+1. As implemented in this project, the modifications to the relevnt `/etc/sudoers.d` files may create sub-optimal SELinux transistions. If so, it will be up to the watchmaker-user to deactivate the `ash-linux.el7.STIGbyID.cat2.RHEL-07-020023` and then provide their own mapping-modifications as a substitute. Deactivation can be done via the `ash-linux:lookup:skip-stigs` list-variable in Pillar.
```

Comment:
Not sure I get your meaning?

Reply:
Oh. That functionality's been present for a _long_ time, now. I sorta take it as a given that people that _need_ to exclude states via the `skip-stigs` method already know how to do so (see: the `ash-linux-formula`'s [`pillar.example`](https://github.com/plus3it/ash-linux-formula/blob/master/pillar.example#L25-L27) file).

Reply:
Ironically, @nortiz34 was using it for at least one of his projects (b/c, some of the `STIGbyID.catN` modules were breaking other stuff that he needed to work more than he needed to check a box on an ACAS report-reply).

Reply:
With the various formulae's `pillar.example` files being wholly within the respective formulaes' GitHub projects, how would we portably-link from the watchmaker doc to the relevant `pillar.example` file-section. I linked _here_ because I don't particularly worry about these PR-comments needing to be portable. About the only portable way I can think to "link" is to _mention_ the project and file rather than actually _link_ to it.

Reply:
Portability isn't much of a concern for these docs. There are plenty of links to external hosts and projects.

Reply:
I just hate dead links when the MDs are moved someplace that the root-URL changes or is otherwise unavailable.

Reply:
Alright. Think I'm going to add a pillar README into the ash-linux-formula project. At the very least, the skip-formulas handler/feature warrants discussion.

Reply:
Submitted  [PR #404](https://github.com/plus3it/ash-linux-formula/pull/404) to the `ash-linux-formula` project. Once that content is merged, can add a link to it as part of this document's notes.

Reply:
Updated to point at ash-linux-formula's [README_PillarContents.md](https://github.com/plus3it/ash-linux-formula/blob/master/README_PillarContents.md#skip-stigs) contents.